src/initialization.js

```diff
@@ -1,42 +1,99 @@
+
 var initialization = {
-    name: 'insertSDKNameHere',
-/*  ****** Fill out initForwarder to load your SDK ******
+    name: 'RoktWsdk',
+    moduleId: 181,
+    /*  ****** Fill out initForwarder to load your SDK ******
     Note that not all arguments may apply to your SDK initialization.
     These are passed from mParticle, but leave them even if they are not being used.
     forwarderSettings contain settings that your SDK requires in order to initialize
     userAttributes example: {gender: 'male', age: 25}
     userIdentities example: { 1: 'customerId', 2: 'facebookId', 7: 'emailid@email.com' }
     additional identityTypes can be found at https://github.com/mParticle/mparticle-sdk-javascript/blob/master-v2/src/types.js#L88-L101
 */
-    initForwarder: function(forwarderSettings, testMode, userAttributes, userIdentities, processEvent, eventQueue, isInitialized, common, appVersion, appName, customFlags, clientId) {
-        /* `forwarderSettings` contains your SDK specific settings such as apiKey that your customer needs in order to initialize your SDK properly */
-
+    initForwarder: function (
+        forwarderSettings,
+        testMode,
+        _userAttributes,
+        _userIdentities,
+        processEvent,
+        eventQueue,
+        _isInitialized,
+        _common,
+        _appVersion,
+        _appName,
+        _customFlags,
+        _clientId
+    ) {
         if (!testMode) {
-            /* Load your Web SDK here using a variant of your snippet from your readme that your customers would generally put into their <head> tags
-               Generally, our integrations create script tags and append them to the <head>. Please follow the following format as a guide:
-            */
+            if (!window.Rokt || !(window.Rokt && window.Rokt.currentLauncher)) {
+                var target = document.head || document.body;
+                var script = document.createElement('script');
+                script.type = 'text/javascript';
+                script.src = roktLauncherScript;
+                script.async = true;
+                script.crossOrigin = 'anonymous';
+                script.fetchPriority = 'high';
+                script.id = 'rokt-launcher';
+
+                script.onload = function () {
+                    // Once the script loads, ensure the Rokt object is available
+                    if (
+                        window.Rokt &&
+                        typeof window.Rokt.createLauncher === 'function' &&
+                        window.Rokt.currentLauncher === undefined
+                    ) {
+                        window.Rokt.createLauncher({
+                            accountId: forwarderSettings.accountId,
+                            sandbox:
+                                forwarderSettings.sandboxMode &&
+                                forwarderSettings.sandboxMode === 'True'
+                                    ? true
+                                    : false,
```

Comment:
This could be simplified to 
```suggestion
                            sandbox: forwarderSettings.sandboxMode === 'True'
```
since if `forwarderSettings.sandboxMode` was `undefined` or `null`, this statement would return `false`